### PR TITLE
Scaffolding identity chloe

### PR DESCRIPTION
**Pull request by chloejonessoftwire**
Removed the register user from the admin login page, updated a tiny bit of styling on the page too, all tests passing. Currently code I plan to delete is commented out so I can show Yvonne what I changed.

### CODE REVIEW

File: Areas/Identity/IdentityHostingStartup.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                services.AddDbContext<WhaleSpottingContext>(options =>
+                    // options.UseSqlServer(
+                    //     context.Configuration.GetConnectionString("WhaleSpottingContextConnection")));
+                    options
+                         .UseNpgsql(Environment.GetEnvironmentVariable("DATABASE_URL")));
+
+
+                services.AddDefaultIdentity<AdminUser>(options => options.SignIn.RequireConfirmedAccount = true)
+                    .AddEntityFrameworkStores<WhaleSpottingContext>();
+                    // .AddUserManager<ApplicationUserManager>()      // <- use only if use have custom implementation
```
Comment (ChristianMcc) [inline Areas/Identity/IdentityHostingStartup.cs]: we can delete this if we don't need it :)

File: Areas/Identity/IdentityHostingStartup.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+[assembly: HostingStartup(typeof(whale_spotting.Areas.Identity.IdentityHostingStartup))]
+namespace whale_spotting.Areas.Identity
+{
+    public class IdentityHostingStartup : IHostingStartup
+    {
+        public void Configure(IWebHostBuilder builder)
+        {
+            builder.ConfigureServices((context, services) => {
+                services.AddDbContext<WhaleSpottingContext>(options =>
+                    // options.UseSqlServer(
```
Comment (ChristianMcc) [inline Areas/Identity/IdentityHostingStartup.cs]: can we delete the comment and also change this to use the ConnectionHelper I've created now like in WhaleSpottingContext.cs please :)

File: appsettings.json [hunk 3]
```diff
@@ -1,20 +1,21 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "DataSource=app.db;Cache=Shared"
+    "DefaultConnection": "DataSource=app.db;Cache=Shared",
+    "WhaleSpottingContextConnection": "Server=(localdb)\\mssqllocaldb;Database=whale-spotting;Trusted_Connection=True;MultipleActiveResultSets=true"
```
Comment (ChristianMcc) [inline appsettings.json]: I don't think this is right, we're not using the connection string here so we can delete this - we're getting our connection string from a .env file instead

File: Areas/Identity/Data/WhaleSpottingContext.cs [hunk 4]
```diff
@@ -0,0 +1,26 @@
+using System;
```
Comment (ChristianMcc) [inline Areas/Identity/Data/WhaleSpottingContext.cs:1]: I'm guessing these were just created for you, it surprises me that we need these as we have our own context files but if it works locally I can go with it!

**Review by ChristianMcc — COMMENTED** [review]
Looking good, hard to tell exactly what is and isn't needed herebut if it's working locally that I'm happy to believe it's good!

File: appsettings.json (hunk 3 above)
Comment (chloejonessoftwire) [reply appsettings.json]: deleted all this, still works :)

File: Areas/Identity/Data/WhaleSpottingContext.cs (hunk 4 above)
Comment (chloejonessoftwire) [reply Areas/Identity/Data/WhaleSpottingContext.cs:1]: the build fails without this file so it is used!

**Review by ChristianMcc — APPROVED** [review]
LGTM :)

**Pull request merged**